ex2/ex2.js

```diff
@@ -8,3 +8,24 @@
  *     fn(",,,", 4) === -1
  *     fn("hel,lowo,rld", 0) - Throws
  */
+function getTheCommaIndexAt(str, commaIndex) {
+
+	if (commaIndex === 0) {
+		throw new RangeError('Throws...');
```

Comment:
The message should be explanatory to the caller: instead of Throws... do `Invalid # of comma`